server/db/index.js

```diff
@@ -1,5 +1,22 @@
 'use strict';
-var db = require('./_db');
+var db = require( './_db' );
 module.exports = db;
+const bluebird = require( 'bluebird' );
 
-var User = require('./models/user');
\ No newline at end of file
+
+var User = require( './models/Organizations/User' );
+var Team = require( './models/Organizations/Team' );
+var Organization = require( './models/Organizations/Organization' );
+var UserOrganization = require( './models/Organizations/UserOrganization' );
+var UserTeam = require( './models/Organizations/UserTeam' );
+
+bluebird.all( [ User, Team, Organization, UserOrganization, UserTeam ] )
+  .spread( ( User, Team, Organization, UserOrganization, UserTeam ) => {
+    User.addAssociations( db );
+    Team.addAssociations( db );
+    Organization.addAssociations( db );
+  } )
+
```

Comment:
the association calls are defined as a class method for each model.
I used bluebird to make sure nothing async was happening.